src/test/java/org/threeten/extra/chrono/TestInternationalFixedChronology.java

```diff
@@ -0,0 +1,1191 @@
+        assertEquals(InternationalFixedChronology.INSTANCE.prolepticYear(InternationalFixedEra.CE,    2),    2);
+        assertEquals(InternationalFixedChronology.INSTANCE.prolepticYear(InternationalFixedEra.CE,    1),    1);
+        assertEquals(InternationalFixedChronology.INSTANCE.prolepticYear(InternationalFixedEra.CE, 2000), 2000);
+        assertEquals(InternationalFixedChronology.INSTANCE.prolepticYear(InternationalFixedEra.CE, 1582), 1582);
+    }
+
+    @Test(expectedExceptions = ClassCastException.class)
+    public void test_prolepticYear_badEra() {
+        InternationalFixedChronology.INSTANCE.prolepticYear(IsoEra.CE, 4);
+    }
+
+    @Test
+    public void test_Chronology_eraOf() {
+        assertEquals(InternationalFixedChronology.INSTANCE.eraOf(0), InternationalFixedEra.CE);
+    }
+
+    @Test(expectedExceptions = DateTimeException.class)
+    public void test_Chronology_eraOf_invalid() {
+        InternationalFixedChronology.INSTANCE.eraOf(2);
+    }
+
+    @Test
+    public void test_Chronology_eras() {
+        List<Era> eras = InternationalFixedChronology.INSTANCE.eras();
+        assertEquals(eras.size(), 1);
+        assertEquals(eras.contains(InternationalFixedEra.CE), true);
+    }
+
+    //-----------------------------------------------------------------------
+    // Chronology.range
+    //-----------------------------------------------------------------------
+    @Test
+    public void test_Chronology_range() {
+        assertEquals(InternationalFixedChronology.INSTANCE.range(ChronoField.DAY_OF_WEEK),           ValueRange.of( 1,   7));
+        assertEquals(InternationalFixedChronology.INSTANCE.range(ChronoField.DAY_OF_MONTH),          ValueRange.of( 1,  28));
+        assertEquals(InternationalFixedChronology.INSTANCE.range(ChronoField.DAY_OF_YEAR),           ValueRange.of( 1, 365, 366));
+        assertEquals(InternationalFixedChronology.INSTANCE.range(ChronoField.MONTH_OF_YEAR),         ValueRange.of( 1,  13));
+        assertEquals(InternationalFixedChronology.INSTANCE.range(ChronoField.ALIGNED_WEEK_OF_MONTH), ValueRange.of( 1,   4));
+        assertEquals(InternationalFixedChronology.INSTANCE.range(ChronoField.PROLEPTIC_MONTH),       ValueRange.of(13, 13_000_000 - 1));
+        assertEquals(InternationalFixedChronology.INSTANCE.range(ChronoField.YEAR_OF_ERA),           ValueRange.of( 1, 999_999));
+        assertEquals(InternationalFixedChronology.INSTANCE.range(ChronoField.YEAR),                  ValueRange.of( 1, 999_999));
+        assertEquals(InternationalFixedChronology.INSTANCE.range(ChronoField.EPOCH_DAY),             ValueRange.of(-719528, 365_000_000));
+    }
+
+    //-----------------------------------------------------------------------
+    // InternationalFixedDate.range
+    //-----------------------------------------------------------------------
+    @DataProvider(name = "ranges")
+    Object[][] data_ranges() {
```

Comment:
1. You don't have to align everything so much (yes, some of the original stuff does some alignment, but not to this degree, and it's a pain to maintain.  Save yourself some trouble)
 2. You're testing things that won't change twice, for no apparent reason (like `PROLEPTIC_MONTH`).  Don't bother.  Just make sure you test for those whose ranges _will_ change.